src/main/scala/com/amazon/deequ/analyzers/KLLSketch.scala

```diff
@@ -73,22 +73,35 @@ object KLLState{
 
 }
 
+/**
+ * Parameter definition for KLL Sketches.
+ * @param sketchSize size of kll sketch
+ * @param shrinkingFactor  shrinking factor of kll sketch
+ * @param numberOfBuckets  number of buckets
+ */
+case class kllParameters(sketchSize: Int, shrinkingFactor: Double, numberOfBuckets: Int)
```

Comment:
Please change the name to `KLLParameters`, classes should always start with capital letters

Reply:
Done.